nvflare/app_common/executors/in_process_client_api_executor.py

```diff
@@ -69,7 +69,7 @@ def __init__(
             raise ValueError(f"invalid task_script_path '{task_script_path}'")
 
         # only support main() for backward compatibility
-        self._task_fn_path = task_script_path.replace(".py", ".main")
```

Comment:
Suggest moving in process api enhancement in another PR

Reply:
Holger asked to include this, so the example will work with the same PR.